package.json

```diff
@@ -31,20 +31,17 @@
     "rebuild-electron": "./node_modules/.bin/electron-rebuild -f -m ./packages/loot-core",
     "rebuild-node": "yarn workspace loot-core rebuild",
     "lint": "cross-env NODE_ENV=development yarn workspaces foreach --verbose run lint --max-warnings 0",
-    "postinstall": "rm -rf ./packages/loot-design/node_modules/react && rm -rf ./node_modules/react-native && patch-package"
+    "postinstall": "rm -rf ./packages/loot-design/node_modules/react && patch-package"
   },
   "devDependencies": {
     "cross-env": "^5.1.5",
     "eslint": "8.35.0",
     "eslint-config-react-app": "7.0.1",
     "eslint-plugin-prettier": "4.2.1",
-    "husky": "^3.0.4",
```

Comment:
Would be really nice to finally get husky + lint-staged (or similar) working again. I see soo many commits that have `lint` step broken. Gets pretty annoying..

(but out of the scope of your PR; just thought it's worth mentioning)

Reply:
FWIW my experience with those tools has been not great — they don’t pick up the path to `node` from GH Desktop in many cases and they slow down the commit process. IMO editor integrations (if people want those) + checks in CI are sufficient

Reply:
It's a question of individual productivity versus maintainer productivity. I'll open a discussion up about what we do here sometime later. For now we can definitely remove husky as it's not used either way.